<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchRenderDialog</class>
 <widget class="QWidget" name="BatchRenderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>View settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_14">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Exported size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="widthEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>The dimensions used when saving.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>x</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="heightEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>The dimensions used when saving.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Supersampling:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sampleSpinBox">
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="aspectCheckBox">
        <property name="text">
         <string>Retain stored aspect ratio</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="maxiterCheckBox">
        <property name="text">
         <string>Limit iterations (approximate):</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="maxiterEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="prevMaxiterCheckBox">
        <property name="text">
         <string>Limit iterations to previously found plus</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="prevMaxiterComboBox">
        <item>
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>150%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200%</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Automatic file naming</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filename pattern (%n for number):</string>
        </property>
        <property name="buddy">
         <cstring>fileTemplateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="fileTemplateEdit">
           <property name="toolTip">
            <string>&lt;p&gt;File name template&lt;/p&gt;&lt;p&gt;This should be a filename, with &quot;%n&quot; somewhere to indicate where the current number should be substituted.&lt;/p&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fileOpenButton">
           <property name="toolTip">
            <string>&lt;p&gt;Choose a file name template&lt;/p&gt;&lt;p&gt;You will need to add '%n&quot; somewhere in the template to indicate the position where the current number should be substituted.&lt;/p&gt;</string>
           </property>
           <property name="text">
            <string>Choose...</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/ClientWindowGui/images/clientwindow/fileopen.png</normaloff>:/ClientWindowGui/images/clientwindow/fileopen.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Files will be stored in a numbered sequence according to the following pattern. It should contain &quot;%n&quot; where the number will be substituted. Leaving it out is possible, it will be automatically appended at the end, likewise for the &quot;.png&quot; suffix. Example: Just &quot;image&quot; will turn into &quot;image0001.png&quot; for the first file.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="overwriteCheckBox">
          <property name="toolTip">
           <string>&lt;p&gt;If checked, the Save button overwrites files without asking.&lt;/p&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Force overwrite</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>widthEdit</tabstop>
  <tabstop>heightEdit</tabstop>
  <tabstop>sampleSpinBox</tabstop>
  <tabstop>aspectCheckBox</tabstop>
  <tabstop>fileTemplateEdit</tabstop>
  <tabstop>fileOpenButton</tabstop>
  <tabstop>overwriteCheckBox</tabstop>
 </tabstops>
 <resources>
  <include location="frac.qrc"/>
 </resources>
 <connections/>
</ui>
